deploy/charts/cert-manager/values.yaml

```diff
@@ -38,6 +38,11 @@ global:
   # The optional priority class to be used for the cert-manager pods.
   priorityClassName: ""
 
+  # Global property to set all your pods to run as root inside the container. Enabled by default in kubernetes 1.33 (https://kubernetes.io/blog/2025/04/25/userns-enabled-by-default/)
```

Comment:
I'm not sure this line is entirely correct, I'd like to think of something else, specifically this part:

> # Global property to set all your pods to run as root inside the container.

This field alone won't do that, but it's a step towards that. Perhaps something on the lines of:
"We use pod namespaces by default, to opt out of this and use the node's namespaces, set this to true." But there is probably a better sentence. Same applied to all the other values below, I'll have a think today.

Reply:
I will probably edit them again and push to get your opinion but you are right. Even the kubetl explain is kinda confusing if you read them the first time.

Reply:
@hawksight do you think the doc should go on all the lines or just the global one should have the detailed one? Also sidenote, I have no idea why the tests seem to be failing.. since nothing in tests are changed per se

Reply:
I think it looks good the way it is now 👍 Thanks for editing.